src/ccache.h

```diff
@@ -136,6 +137,7 @@ bool hash_file(struct mdfour *md, const char *fname);
 void cc_log(const char *format, ...) ATTR_FORMAT(printf, 1, 2);
 void cc_bulklog(const char *format, ...) ATTR_FORMAT(printf, 1, 2);
 void cc_log_argv(const char *prefix, char **argv);
+bool cc_copylog(const char *path);
```

Comment:
Perhaps rename to something like `cc_dump_log_buffer`?

Reply:
Done